ui/templates/browse.html

```diff
@@ -6,13 +6,72 @@
 <img src="{{.LogoPath}}" alt="." height="92" width="60" align="right" />
 <img src="{{.LogoPath}}" alt="istio logo" height="123" width="80" align="right" />
 <h1>Φορτίο (fortio) v{{.Version}}</h1>
+<script src="{{.Version}}/static/js/fortio_chart.js"></script>
+<script>
```

Comment:
Feels like this could be another js file. But feel free to move it in later PRs.

Reply:
👍